ArticleTemplates/assets/js/modules/quiz.js

```diff
@@ -466,8 +465,8 @@ function showScore() {
     initPositionPoller();
 
     // Scroll score panel into view
-    const scroll = new SmoothScroll();
-    scroll.animateScroll('#quiz-scores', null, {speed: 1500, offset: 40});
+    const scoresElement = document.getElementsByClassName('quiz-scores')[0]
```

Comment:
The syntax previously used to select this suggests it's an `id` (the `#` tends to signify that), but you've switched to selecting it via class name. Are both present on the element?

Reply:
Ah I was copying line 464 and not engaging brain. This way might work but I'll switch to `getElementById` so it's an exact replacement for `#quiz-scores` :+1: